packages/rx_bloc/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [6.0.0]
```

Comment:
Make sure the rx_bloc_cli accommodates the breaking change of the rx_bloc eco system